polkadot-protocol/protocol-components/transactions/transactions-formats.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Transactions Formats
+description: TODO
+---
+
+# Transactions Formats
+
+## Introduction
+
+This article describes the data structure of signed and unsigned transactions in Polkadot SDK-based chains. This is particularly useful for understanding how the transaction pool checks incoming transactions. Parachain builders will find this helpful in customizing how their transactions are formatted and writing client applications that must adhere to a chosen format.
+
+Extrinsics normally contain a signature, some data to describe whether the extrinsic has passed some validity checks and a reference to the pallet and call that it is intended for. This format provides a way for applications to ensure that the requirements for an extrinsic are met and correctly constructed.
+
+- Unchecked - signed transactions requiring some validation check before being accepted in the transaction pool. Any unchecked extrinsic contains the signature for the sent data and some extra data
+- Checked - inherent extrinsic, which, by definition, don't require signature verification. Instead, they carry information on where the extrinsic comes from and some extra data
+- Opaque - used for cases when an extrinsic hasn't yet been committed to a format but can still be decoded
+
+Extra data can be any additional information helpful to attach to a transaction or inherent. For example, the nonce of the transaction, the tip for the block author, or how long the extrinsic is valid for. This information is provided by a specialized extensions that help determine the validity and ordering of extrinsics before they get included in a block.
+
+A signed transaction might be constructed like so:
+
+```rust
+--8<-- "code/polkadot-protocol/transactions/transactions-formats/signed-tx-example.rs"
+```
+
+## How Transactions are Constructed
+
+Polkadot SDK defines its transaction formats generically to allow developers to implement custom ways to define valid transactions. In a runtime built with FRAME, however (assuming transaction version 4), a transaction must be constructed by submitting the following encoded data:
+
+`<signing account ID> + <signature> + <additional data>`
+
+When submitting a signed transaction, the signature is constructed by signing:
+
+- The actual call is composed of the following:
```

Comment:
This is fine for now, but maybe a mermaid chart for stuff like this in the future 👀